semantic_conventions/resource/faas.yaml

```
@@ -16,30 +16,38 @@ groups:
           [`code.namespace`/`code.function`](../../trace/semantic_conventions/span-general.md#source-code-attributes)
           span attributes).
 
-          On Azure, an Azure function app can host multiple functions that would usually share
-          a TracerProvider, so this MUST be set as a span attribute instead on Azure. See also
-          `faas.id`. Additionally, the full name `<FUNCAPP>/<FUNC>` MUST be used for this attribute, i.e., function app name followed by a forward slash followed by the function
-          name (this form can also be seen in resource JSON for the function).
-        examples: ['my-function']
+          For some cloud providers, the above definision is ambiguous. The following
```

```suggestion
          For some cloud providers, the above definition may be ambiguous. The following
```

---

```suggestion
          For some cloud providers, the above definition is ambiguous. The following
```

It definitely is ambiguous for Azure. We already have "some" so there is no need to add "may".